webapp/src/app.tsx

```diff
@@ -3924,6 +3926,8 @@ export class ProjectView
                     </div>
                 </div>
                 <div id="maineditor" className={(sandbox ? "sandbox" : "") + (inDebugMode ? "debugging" : "")} role="main" aria-hidden={inHome}>
+                    {!(isSidebarTutorial && flyoutOnly) && inTutorial && <tutorial.TutorialCard ref={ProjectView.tutorialCardId} parent={this} pokeUser={this.state.pokeUserComponent == ProjectView.tutorialCardId} />}
```

Comment:
ah dang i was hoping that would be related cus I couldn't figure out why it changed XD maybe it is related -- i will try it out

I think I thought it would have to be here because I did it at the beginning of making this PR and wasn't sure how I was gonna position it and thought it would have to be this way. I'm gonna try putting it back where it was before and see!